packages/mobile/src/geth/network-config.ts

```diff
@@ -5,21 +5,26 @@ export default {
   [Testnets.integration]: {
     nodeDir: `.${Testnets.integration}`,
     syncMode: SYNC_MODE_ULTRALIGHT,
+    blockchainApiUrl: 'https://integration-dot-celo-testnet.appspot.com/',
   },
   [Testnets.alfajoresstaging]: {
     nodeDir: `.${Testnets.alfajoresstaging}`,
     syncMode: SYNC_MODE_ULTRALIGHT,
+    blockchainApiUrl: 'https://alfajoresstaging-dot-celo-testnet.appspot.com/',
   },
   [Testnets.alfajores]: {
     nodeDir: `.${Testnets.alfajores}`,
     syncMode: SYNC_MODE_ULTRALIGHT,
+    blockchainApiUrl: 'https://pilot-dot-celo-testnet-production.appspot.com/',
```

Comment:
alfajores?

Reply:
Good catch. Should have been `https://alfajores-dot-celo-testnet-production.appspot.com/`
Fixed.